fix/anko.go

```diff
@@ -12,30 +13,35 @@ func Anko(content string) (string, error) {
 	bb := &bytes.Buffer{}
 
 	lines := strings.Split(content, "\n")
+	l := len(lines)
 
-	// fix create_table
-	inCreateTable := false
-	for i := 0; i < len(lines); i++ {
+	for i := 0; i < l; i++ {
 		line := lines[i]
 		tl := strings.TrimSpace(line)
 		if strings.HasPrefix(tl, "create_table") {
-			line = strings.Replace(line, ", func(t) {", ") {", -1)
-			inCreateTable = true
-		}
-		if strings.HasPrefix(tl, "}") && inCreateTable {
-			inCreateTable = false
-		}
-		if strings.HasPrefix(tl, "})") && inCreateTable {
-			line = "}"
-			inCreateTable = false
-		}
-		lines[i] = line
-	}
-
-	// fix raw
-	for i, line := range lines {
-		tl := strings.TrimSpace(line)
-		if strings.HasPrefix(tl, "raw(") {
+			// skip already converted create_table
+			if strings.Contains(line, ", func(t) {") {
+				// fix create_table
+				line = strings.Replace(line, ", func(t) {", ") {", -1)
```

Comment:
Since you are re-doing this, it would be nice to use regex.ReplaceAllString() so that it'll not care about any whitespace. (We didn't have `, func`, but `,func` and it wouldn't work.

Reply:
Done. :)